src/ontology/uberon-edit.obo

```diff
@@ -64352,7 +64349,6 @@ consider: MESH:A11.284.195.190.750.510
 consider: NCIT:C13255
 consider: NIF_Subcellular:sao2006047981
 consider: UMLS:C0025979 {source="ncithesaurus:Actin_Filament"}
-consider: Wikipedia:Microfilament
```

Comment:
Could these Wikipedia references be in another annotation, like xref, for terms without a definition?

Reply:
Yes, I suppose they could be xrefs or seeAlso. Is that really necessary, though? We’re talking about _obsolete_ terms after all. The only reason someone is ever going to look at them is they have been using them in their own ontologies or databases and they need to know how to replace them – and for that they will be looking at `consider` tags, not xrefs or anything else.

Reply:
We never know. If we can keep the information, it's always better.

Reply:
Fine, if it’s make you happy.

I believe that keeping information for the sake of keeping information utterly ridiculous, though. There’s a difference between “keeping information” and keeping all the crap that we don’t dare clean up. The focus should be on providing useful information, which sometimes means _removing_ stuff.